src/main/java/be/seeseemelk/mockbukkit/scoreboard/ScoreboardMock.java

```diff
@@ -37,9 +55,8 @@ public ObjectiveMock getObjective(String name) throws IllegalArgumentException
 	@Override
 	public Set<Objective> getObjectivesByCriteria(String criteria) throws IllegalArgumentException
 	{
-		return objectives.values().stream()
-		       .filter(objective -> objective.getCriteria().equals(criteria))
-		       .collect(Collectors.toSet());
+		return objectives.values().stream().filter(objective -> objective.getCriteria().equals(criteria))
+				.collect(Collectors.toSet());
```

Comment:
This is much harder to read imo

Reply:
I agree.
Didn't change that though, probably my auto fornatter.
We really need to get the formatter workflow running again.